### PR TITLE
Add phplist as hard

### PR DESCRIPTION
**Pull request by xplosionmind**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -10072,6 +10072,17 @@
             "meethue.com",
             "account.meethue.com"
         ]
+    
+    },
```
Comment (tupaschoal) [inline _data/sites.json:10076]:
```suggestion

    },
```

File: _data/sites.json [hunk 2]
```diff
@@ -10072,6 +10072,17 @@
             "meethue.com",
             "account.meethue.com"
         ]
+
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json:10077]:
```suggestion
    {
```

File: _data/sites.json [hunk 3]
```diff
@@ -10072,6 +10072,17 @@
             "meethue.com",
             "account.meethue.com"
         ]
+
+    },
+    {
```
Comment (tupaschoal) [inline _data/sites.json:10077]:
```suggestion
    },

    {
```

File: _data/sites.json [hunk 4]
```diff
@@ -10072,6 +10072,18 @@
...
         ]
+
+    },
+
+    {
+        "name": "phplist",
+        "url": "https://www.phplist.com/privacy",
+        "difficulty": "hard",
+        "notes": "Write an email to the email address in the link, requesting all data to be deleted.",
+        "notes_it": "Scrivere un’email all’indirizzo email indicato nella pagina del link, chiedendo che tutti i dati siano eliminati.",
```
Comment (tupaschoal) [inline _data/sites.json:10083]: We can provide an email directly on the website, we would just need to do this:


```suggestion
        "notes": "Write an email requesting all data to be deleted.",
        "notes_it": "Scrivere un’email chiedendo che tutti i dati siano eliminati.",
        "email": "email@domain.com",
```

 (I can't see what the correct email is at the moment)

Comment (xplosionmind) [reply _data/sites.json:10083]: the email is `info@phplist.com`
Screen Shot 2022-09-25 at 11 41 38 AM

**Review by xplosionmind — COMMENTED** [review]
by adding the email address I noted, everything should be correct and ready.

File: _data/sites.json [hunk 5]
```diff
@@ -10072,6 +10072,18 @@
             "meethue.com",
             "account.meethue.com"
         ]
+
+    },
```
Comment (tupaschoal) [inline _data/sites.json:10076]:
```suggestion
    },
```

**Review by tupaschoal — COMMENTED** [review]
Content-wise it is ok, though it still needs to respect the sort order in the JSON and come after PhishTank and Photobucket. Can you move it, please?

**Comment by tupaschoal** [comment]
> I am sorry, I am a mess with GitHub. Could you move it yourself? Thanks a lot.

Done

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Comment by xplosionmind** [comment]
> Thanks for your contribution!

Thank *you* for fixing my mess!  
Have a nice day.

**Pull request merged**